src/SourceBuild/content/eng/build.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding(PositionalBinding=$false)]
```

Comment:
I wonder if at some point it will be possible to simply use the common arcade build script.

Reply:
Let me file an issue for that. It would be good, yes.

Reply:
https://github.com/dotnet/source-build/issues/3964